src/application/services/useHeader.ts

```diff
@@ -0,0 +1,82 @@
+import type { ComputedRef } from 'vue';
+import { computed, onMounted, ref, watch } from 'vue';
+import { useRoute } from 'vue-router';
+import { AppStateController } from '@/domain';
+import type { Page, PageList } from '@/domain/entities/Page';
+import type { TabList } from '@/domain/entities/Tab';
+import { workspaceService } from '@/domain/index';
+
+interface useHeaderComposableState {
+  getOpenedPages: () => void;
```

Comment:
seems redundant. `tabs` is enough

Reply:
we use onMounted() that loads all pages using this method, this is redundant too?